src/templates/app/index.html

```diff
@@ -217,7 +217,7 @@
           <dl class="app-metadata-categories">
             <dt>{{ _('Categories') }}</dt>
             {% for category in CATEGORIES %}
-              {% if indexOf(this.categories, category.slug) != -1 %}
+              {% if indexOf(CATEGORIES, category.slug) != -1 %}
```

Comment:
Yep you're right I missed the context here...was looking at something else earlier.